github-helper.js

```diff
@@ -0,0 +1,70 @@
+exports.GithubHelper = class {
+  constructor({ client, context }) {
+    this.client = client;
+    this.context = context;
+  }
+
+  async assignToProjects({ columnName, newProjectNames }) {
+    const allProjects = await this.fetchProjects();
+    const columnIds = allProjects
+      .map((project) => project.node) // remove top-level "node" attribute
+      .filter(({ name: projectName }) => newProjectNames.includes(projectName))
+      .map(({ columns: { edges: columns } }) => (
+        columns.find((column) => column.node.name === columnName)
+          .node
+          .databaseId
+      ));
+    await this.addProjectCards({ columnIds });
+  }
+
+  async fetchProjects({ cursor } = {}) {
```

Comment:
I don't see the benefit in using named arguments for functions with one argument, it's just more curly brackets:

```suggestion
  async fetchProjects(cursor) {
```